app/uk/gov/hmrc/agentaccesscontrol/controllers/AuthorisationController.scala

```diff
@@ -48,13 +51,19 @@ class AuthorisationController @Inject() (override val auditService: AuditService
   }
 
   def isAuthorisedForPaye(agentCode: AgentCode, empRef: EmpRef) = Action.async { implicit request =>
-    authorisationService.isAuthorisedForPaye(agentCode, empRef) map {
-      case authorised if authorised => Ok
-      case notAuthorised => Unauthorized
-    }
+
+    val payeEnabled : Boolean = configuration.getBoolean("features.allowPayeAccess").getOrElse(false)
+
+    if(payeEnabled){
```

Comment:
formatting: please add a space between ) and { (as per IntelliJ defaults)
I'd probably also move the else onto its own line and probably put it in braces